homework/index.js

```diff
@@ -30,15 +33,189 @@
+            id: `${item.login}Name`,
+            class: 'cont-name',
+            text: item.login,
+            href: item.html_url,
+            target: '_blank',
+          });
+          createAndAppend('div', contributorDiv, {
+            id: `${item.login}Badge`,
+            class: 'cont-badge',
+            text: item.contributions,
+          });
+        });
+      },
+      err => {
+        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
+      },
+    );
+  }
+
+  function updateContributorSection(selected) {
+    const contributorsDiv = document.getElementById('contsDiv');
+    contributorsDiv.innerHTML = '';
+    createAndAppend('p', contributorsDiv, { id: 'cont-header', text: 'Contributors' });
+    const contributorsLists = createAndAppend('ul', contributorsDiv, { id: 'conts-list' });
+
+    fetchJSON(`https://api.github.com/repos/HackYourFuture/${selected}/contributors`).then(
+      data => {
+        data.forEach(item => {
+          const contributorListItem = createAndAppend('li', contributorsLists, {
+            id: `${item.login}ListItem`,
+            class: 'cont-li',
+          });
+
+          createAndAppend('img', contributorListItem, {
+            id: `${item.login}Img`,
+            class: 'cont-avatar',
+            src: item.avatar_url,
+          });
+
+          const contributorDiv = createAndAppend('div', contributorListItem, {
+            id: `${item.login}Div`,
+          });
+
+          createAndAppend('a', contributorDiv, {
+            id: `${item.login}Name`,
+            class: 'cont-name',
+            text: item.login,
+            href: item.html_url,
+            target: '_blank',
+          });
+          createAndAppend('div', contributorDiv, {
+            id: `${item.login}Badge`,
+            class: 'cont-badge',
+            text: item.contributions,
+          });
+        });
+      },
+      err => {
+        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
+      },
+    );
+  }
+
+  function updateInfoSection(selected, data) {
+    const selectedData = getSelectedData(selected, data);
+
+    const repoName = document.getElementById('repoName');
+    repoName.innerText = '';
+    repoName.innerText = `Repository: ${selectedData.name}`;
+    repoName.href = selectedData.html_url;
+    repoName.target = '_blank';
+
+    const desc = document.getElementById('desc');
+    desc.innerText = '';
+    desc.innerText = `Description: ${selectedData.description}`;
+
+    const forks = document.getElementById('forks');
+    forks.innerText = '';
+    forks.innerText = `Forks: ${selectedData.forks}`;
+
+    const updatedAt = document.getElementById('updated_at');
+    updatedAt.innerText = '';
+    updatedAt.innerText = `Updated: ${selectedData.updated_at}`;
+  }
+
+  function main(url) {
+    fetchJSON(url).then(
+      data => {
+        const repoName = data.map(repo => repo.name).sort((a, b) => a.localeCompare(b));
```

Comment:
Umm.. maybe you can add the ID being returned in the `data` by the API here and send it to `generateSelections` method so that you don't have to write the `id` logic there.

Something like:
```
const repoName = data
  .map(repo => ({
    name: repo.name,
    id: repo.id
  })
  .sort((a,b) => a.name.localeCompare(b.name));
```